network/components/virt_rx.c

```diff
@@ -14,11 +14,11 @@
 #define DRIVER_CH 0
 #define CLIENT_CH 1
 
+#define BROADCAST_ID (NUM_CLIENTS + 1)
```

Comment:
I think that the `BROADCAST_ID` should be fixed because otherwise the channel ID of the ARP depends on how many clients are on the system.

The `CLIENT_CH` should then change to 2 and `BROADCAST_ID` should be 1.

Reply:
Whoops nevermind I misunderstood.